source/bits/lshiftl.ts

```diff
@@ -0,0 +1,21 @@
+import { Bit, Bits } from '../types'
+
+/**
+ * Logical Shift Left
+ * 
+ * Shifts all given bits to left and returns the resulting bits
+ *
+ * @example
+ * lshiftl([1,0,1,1,0,1]) => [0,1,1,0,1,0]
+ *
+ * @param {Array} bits input data
+ * @return {Array} [LSHIFTL bits]
+ */
+export default (bits: Bits): Bits => {
+    const result: Bits = []
+
+    for (let i: number = 0; i < bits.length - 1; i++) result.push(bits[i + 1])
+    result.push(<Bit>0)
```

Comment:
```suggestion
    result.push(0 as Bit)
```

`<Bit>` cast syntax is deprecated AFAIK

Reply:
done